docs/services/bmc_dm.rst

```diff
@@ -1,14 +1,57 @@
 Boston Deformable Mirror
 ========================
+This service operates a Boston Micromachines MEMS DM. The following Boston DMs have been tested with catkit2 thus far:
```

Comment:
This is actually a service for a pair of Boston DMs controlled by the same driver. Since this can be a limitation in some cases, @ehpor and I started working on a new service that is more flexible in this regard in #156. While this will not alter this service here, it might be good to explain that it is for a pair of the same type of Boston DMs, with the same number of actuators, controlled by a single driver.

I will make sure to highlight the respective changes when I write the service doc page in #156.